woocommerce-services.php

```diff
@@ -647,8 +645,7 @@ public function attach_hooks() {
 			$schemas = $schemas_store->get_service_schemas();
 
 			if ( $schemas ) {
-				add_filter( 'woocommerce_shipping_methods', array( $this, 'woocommerce_shipping_methods' ) );
-				add_action( 'woocommerce_load_shipping_methods', array( $this, 'woocommerce_load_shipping_methods' ) );
+				add_filter( 'woocommerce_shipping_methods', array( $this, 'woocommerce_shipping_methods' ), 9999 );
```

Comment:
Why `9999`?

Reply:
Just so the filter is executed after all other shipping methods have been loaded. Would `PHP_INT_MAX` for example make it clearer?

Reply:
The `9999` thing got me wondering what other filters we could use that would remove the need for setting priorities manually.

Do you think using `woocommerce_shipping_method_title` would work? I found it in the [list of WC hooks](https://docs.woocommerce.com/wc-apidocs/hook-docs.html).

Here's some reasoning on why I went digging:
The way I'm reading the WC core code, there's a way that even if we're the last filter, it may not work in some cases is because `woocommerce_shipping_methods` is used in `get_shipping_method_class_names` (https://docs.woocommerce.com/wc-apidocs/source-class-WC_Shipping.html#131). `get_shipping_method_class_names` is  used in `load_shipping_methods` (https://docs.woocommerce.com/wc-apidocs/source-class-WC_Shipping.html#156) where more shipping methods can be loaded after using `woocommerce_load_shipping_methods`, so we may not be getting all of them even if we're the last ones using the filter. It's a bit edgecase, so if another filter doesn't work, I think we can go with what you already have here.